fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -5000,9 +5009,9 @@ class FileBackupAgentImpl {
 						state Reference<IBackupContainer> bc;
 						state TimestampedVersion latestRestorable;
 
-						wait(
-						    store(latestRestorable, getTimestampedVersion(tr, config.getLatestRestorableVersion(tr))) &&
-						    store(bc, config.backupContainer().getOrThrow(tr)));
+						wait(store(latestRestorable, getTimestampedVersion(tr, config.getLatestRestorableVersion(tr))));
+						Reference<IBackupContainer> _bc = wait(config.backupContainer().getOrThrow(tr));
+						bc = fileBackup::getBackupContainerWithProxy(_bc);
```

Comment:
The original `wait` waits for 2 things together, which is more efficient and preferred. Instead of `wait` here, i think you can just add `bc = fileBackup::getBackupContainerWithProxy(bc);` with the original `wait` above.

Reply:
I add another var just in case there is surprise from flow, i can change it to the way you described as well, let me know